LibreNMS/OS/Edgecos.php

```diff
@@ -41,6 +41,10 @@ public function discoverProcessors()
     {
         $device = $this->getDevice();
 
+        if (starts_with($device['sysObjectID'], '.1.3.6.1.4.1.259.10.1.1.')) { //ECS4610
```

Comment:
Judging by this and the other lines in here would we not be better replacing them all with:

`$oid = preg_replace('/.[0-9]+$/', '.1.39.2.1.0', $device['sysObjectID']);`

```
php > $device['sysObjectID'] = '.1.3.6.1.4.1.259.10.1.24.103';
php > $oid = preg_replace('/.[0-9]+$/', '.1.39.2.1.0', $device['sysObjectID']);
php > print_r($oid);
.1.3.6.1.4.1.259.10.1.24.1.39.2.1.0
```